erpnext/accounts/doctype/account/account.py

```diff
@@ -89,7 +89,7 @@ def validate_root_details(self):
 				throw(_("Root cannot be edited."), RootNotEditable)
 
 		if not self.parent_account and not self.is_group:
-			frappe.throw(_("Root Account must be a group"))
+			frappe.throw(_("Root Account must be a group for {0}".format(frappe.bold(self.name))))
```

Comment:
frappe.throw(_("Root Account {0} must be a group".format(frappe.bold(self.name))))